contracts/liquidator/Liquidator.sol

```diff
@@ -1,4 +1,4 @@
-//SPDX-License-Identifier: Unlicense
+// SPDX-License-Identifier: BUSL-1.1
```

Comment:
Hmm, I wonder if we actually want this to be the same license as everything else 🤔

Reply:
🤖 suggesting MIT

Reply:
updated